mm2src/coins/qrc20.rs

```diff
@@ -1537,12 +1536,10 @@ pub struct Qrc20FeeDetails {
 }
 
 async fn qrc20_withdraw(coin: Qrc20Coin, req: WithdrawRequest) -> WithdrawResult {
-    let to_addr = UtxoAddress::from_str(&req.to)
-        .map_err(|e| e.to_string())
+    let to_addr = UtxoAddress::from_legacyaddress(&req.to, &coin.as_ref().conf.address_prefixes)
         .map_to_mm(WithdrawError::InvalidAddress)?;
     let conf = &coin.utxo.conf;
-    let is_p2pkh = to_addr.prefix == conf.pub_addr_prefix && to_addr.t_addr_prefix == conf.pub_t_addr_prefix;
-    if !is_p2pkh {
+    if !to_addr.is_p2pkh() {
         let error = "QRC20 can be sent to P2PKH addresses only".to_owned();
```

Comment:
Maybe we should change this message also to be `p2pkh` and `p2wpkh`

Reply:
It seems that qrc20 cannot go to segwit addresses and this is what this check is supposed to do.
Maybe my is_p2pkh() is confusing and better to return true only for p2pkh (not p2pkh or p2wpkh)